front/stores/appStore.js

```diff
@@ -1,16 +1,9 @@
 import { defineStore } from 'pinia'
-import { StorageSerializers, useLocalStorage } from '@vueuse/core'
-import * as LanguageConstants from '~/constants/LanguageConstants'
-import DateUtils from '~/utils/DateUtils'
-import { FORM_CONSTANTS_TRANSACTION_FIELDS_LIST } from '~/constants/FormConstants'
+import { useLocalStorage } from '@vueuse/core'
 import ResponseUtils from '~/utils/ResponseUtils'
 import { compareVersionStrings } from '~/utils/DataUtils'
 import InfoRepository from '~/repository/InfoRepository.js'
 import { get } from 'lodash'
-import { HERO_ICONS, HERO_ICONS_LIST } from '~/constants/TransactionConstants.js'
-import { NUMBER_FORMAT } from '~/utils/MathUtils.js'
-import ProfileRepository from '~/repository/ProfileRepository'
```

Comment:
There is something very wrong with imports. Without deleting this line Pico wouldn't load for me saying "Cannot access 'BaseRepository' before initialization". Same happens if I remove some of the unused imports in other files...